blocks/market-commentary/market-commentary.js

```diff
@@ -25,10 +25,10 @@ function createMarketCommentaryCard(cardData, placeholders) {
         div(
           { class: 'info' },
           p(
-            span(`${placeholders.powerby}`),
+            span(`${blockConfig.powerby}`),
           ),
           p(
-            span(`${placeholders.publishedon} ${NEWS_DATE}`),
+            span(`${blockConfig.publishedon} ${NEWS_DATE}`),
```

Comment:
@ravverma this is repeated in many blocks , IMO we should take it from placeholder

Reply:
same as above